<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="feed">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>870</width>
     <height>760</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>231</width>
     <height>751</height>
    </rect>
   </property>
   <property name="title">
    <string>Attendance Record</string>
   </property>
   <widget class="QLabel" name="EmpID_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EmpID</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmpID_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EmpID</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="EmpID_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>560</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EmpID</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="EmpPhoto_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>EmpPhoto</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmpPhoto_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>EmpPhoto</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmpName_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EmpName</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="EmpID_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EmpID</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmpID_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>680</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EmpID</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmpID">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EmpID</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmpName_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EmpName</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="EmpName_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>650</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EmpName</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="EmpName_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EmpName</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmpPhoto_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>EmpPhoto</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmpName_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EmpName</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmpName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EmpName</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmpPhoto_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>EmpPhoto</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmpPhoto">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>EmpPhoto</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="EmpPhoto_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>EmpPhoto</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
